src/main/java/org/folio/rest/impl/LoginAPI.java

```diff
@@ -341,9 +571,9 @@ public void postAuthnLogin(String userAgent, String xForwardedFor,
             PostgresClient.getInstance(vertxContext.owner(), tenantId).get(
                 TABLE_NAME_CREDENTIALS, Credential.class, new Criterion(useridCrit),
                 true, getReply-> {
+                  // TODO And now we're actually getting a user's credentials.
                   if (getReply.failed()) {
-                    logger.error("Error in postgres get operation: " +
-                        getReply.cause().getLocalizedMessage());
+                    logger.error("Error in postgres get operation: " + getReply.cause().getLocalizedMessage());
```

Comment:
If you touch it you should replace string concatenation with format specifier:

```
                    logger.error("Error in postgres get operation: {}", getReply.cause().getLocalizedMessage());
```

Reply:
Ok done